repo/blob/s3/s3_storage_test.go

```diff
@@ -526,18 +509,26 @@ func createClient(tb testing.TB, opt *Options) *minio.Client {
 	return minioClient
 }
 
-func createBucket(tb testing.TB, opt *Options) {
+func getOrCreateBucket(tb testing.TB, opt *Options) {
 	tb.Helper()
 
 	minioClient := createClient(tb, opt)
 
-	makeBucket(tb, minioClient, opt, false)
+	getOrMakeBucket(tb, minioClient, opt, false)
 }
 
-func makeBucket(tb testing.TB, cli *minio.Client, opt *Options, objectLocking bool) {
+func getOrMakeBucket(tb testing.TB, cli *minio.Client, opt *Options, objectLocking bool) {
 	tb.Helper()
 
-	if err := cli.MakeBucket(context.Background(), opt.BucketName, minio.MakeBucketOptions{
+	ctx := context.Background()
+
+	if loc, err := cli.GetBucketLocation(ctx, opt.BucketName); err == nil {
+		tb.Log("found bucket", opt.BucketName, "in location", loc)
+
+		return
+	}
+
+	if err := cli.MakeBucket(ctx, opt.BucketName, minio.MakeBucketOptions{
```

Comment:
nit; requesting a one-liner here reasoning why we are doing a get-bucket instead of handling status-exists from make-bucket.